c4_set_parameters.m

```diff
@@ -77,11 +78,26 @@
 fit_min_log_nciv     = 12.5;                   % range of column density samples    [cm⁻²]
 fit_max_log_nciv     = 15.6;                   % from fit to log PDF
 extrapolate_min_log_nciv = 12.5;               % normalization range for the extrapolated region
+% model prior parameters
+
+prior_z_qso_increase = kms_to_z(30000);       % use QSOs with z < (z_QSO + x) for prior
+
+% instrumental broadening parameters
+width = 3;                                    % width of Gaussian broadening (# pixels)
+pixel_spacing = 1e-4;                         % wavelength spacing of pixels in dex
 
+% DLA model parameters: absorber range and model
+num_lines = 3;                                % number of members of the Lyman series to use
 
-% I removed these functions since my code should work without them just in this stage
-% maybe add them later in the final version
+max_z_cut = kms_to_z(3000);                   % max z_DLA = z_QSO - max_z_cut
+max_z_c4 = @(wavelengths, z_qso) ...         % determines maximum z_DLA to search
+    (max(wavelengths) / civ_2_wavelength - 1) - max_z_cut;
 
+min_z_cut = kms_to_z(3000);                   % min z_DLA = z_Ly∞ + min_z_cut
+min_z_c4 = @(wavelengths, z_qso) ...         % determines minimum z_DLA to search
+    max(min(wavelengths) / civ_2_wavelength - 1,                          ...
+        observed_wavelengths(civ_1_wavelength, z_qso) / civ_2_wavelength - 1 + ...
```

Comment:
I think the min_z_c4 should be lower?
The model range is 1410 ~ 1550, but you only search 1548~1550 for the absorbers.

suggested:
```matlab
        observed_wavelengths(1410, z_qso) / civ_2_wavelength - 1 + ...
```

Reply:
That's right